main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       bomb
+//  action 1:   the bomb explodes
+bomb.detonate();
+//  action 2:   the bomb arms
+bomb.arm();
+//  action 3:   the bomb disarms
+bomb.disarm();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       phone
+//  action 1:   the phone dials the number on display
+phone.callNumber();

```

Comment:
this pseudo-code should say `phone.dialDisplayedNumber()`;
The goal is to match the pseudo-code with the plain-english expression. 
- [ ] revise